docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -30,6 +30,10 @@ The major features of C# 1.0 included:
 - [Attributes](../programming-guide/concepts/attributes/index.md)
 - Literals
 
+## C# version 1.2
+
+C# version 1.2 shipped with Visual Studio 2003. It contained a few small fixes to the language. Most notable is that starting with this version, the code generated in a `foreach` loop called <xref:System.IDisposable.Dispose%2A> on an <xref:System.Collections.IEnumerator> when that <xref:System.Collections.IEnumerator> implemented <xref:System.IDisposable>.
```

Comment:
Is fixes the right word -- it implies a bug fix. I'd consider calling Dispose an enhancement or addition rather than a fix, since you can call Dispose yourself when finished with the enumerator.

Reply:
I agree, @rpetrusha.  The `IDisposable` stuff is an enhancement.